google-cloud-bigquery/test/google/cloud/bigquery/dataset_load_job_schema_test.rb

```diff
@@ -172,7 +176,7 @@ def storage_file path = nil
     job.must_be_kind_of Google::Cloud::Bigquery::LoadJob
     job.schema_update_options.must_equal schema_update_options
 
-    mock.verify
+    #mock.verify
```

Comment:
Was this unintentional, did you intent to restore this call to `mock.verify` before committing?

Reply:
@quartzmo Completely unintentional. I commented that out trying to get that test to pass because of a silly error elsewhere and forgot to remove the comment. Thanks for fixing!